### PR TITLE
Mark various APIs in Deno object as unstable

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: cli/js/lib.deno_runtime.d.ts [hunk 1]
```diff
@@ -7,12 +7,12 @@
 /// <reference lib="esnext" />
 
 declare namespace Deno {
-  // @url js/os.d.ts
```
Comment (bartlomieju) [inline cli/js/lib.deno_runtime.d.ts]: I don't think those should be removed

File: cli/js/lib.deno_runtime.d.ts [hunk 2]
```diff
@@ -188,31 +195,39 @@ declare namespace Deno {
    */
   export function execPath(): string;
 
-  // @url js/dir.d.ts
```
Comment (bartlomieju) [inline cli/js/lib.deno_runtime.d.ts]: ditto

File: cli/js/lib.deno_runtime.d.ts [hunk 3]
```diff
@@ -188,31 +195,39 @@ declare namespace Deno {
...
+
   /**
+   * UNSTABLE: maybe needs permissions.
+   *
    * `chdir()` Change the current working directory to path.
    * throws `NotFound` exception if directory not available
    */
   export function chdir(directory: string): void;
 
-  // @url js/io.d.ts
```
Comment (bartlomieju) [inline cli/js/lib.deno_runtime.d.ts]: ditto

File: cli/js/lib.deno_runtime.d.ts [hunk 4]
```diff
@@ -593,102 +630,128 @@ declare namespace Deno {
    *
    *       const tempDirName0 = await Deno.makeTempDir();
    *       const tempDirName1 = await Deno.makeTempDir({ prefix: 'my_temp' });
+   *
+   * Requires allow-write.
    */
+  // TODO(ry) Doesn't check permissions.
   export function makeTempDir(options?: MakeTempDirOptions): Promise<string>;
 
-  // @url js/chmod.d.ts
```
Comment (bartlomieju) [inline cli/js/lib.deno_runtime.d.ts]: ditto

File: cli/js/lib.deno_runtime.d.ts (hunk 1 above)
Comment (ry) [reply cli/js/lib.deno_runtime.d.ts]: What purpose does it have?

Comment (bartlomieju) [reply cli/js/lib.deno_runtime.d.ts]: I'm not sure, but they might be hints for source map, if CI goes green without them then I don't care

Comment (ry) [reply cli/js/lib.deno_runtime.d.ts]: I think it's just cruft from the autogen.

**Pull request merged**